templates/2_Cluster_Pixels.ipynb

```diff
@@ -793,21 +793,22 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    for pixel_fov in subset_pixel_fovs:
</pre><p>What's the point of the subset_pixel_fovs variable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/816/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Looks like an artifact from a previous update, I'll remove this.</p>

Reply:
<p>Actually, scratch that. I'll remove <code>pixel_fovs</code> and just use <code>subset_pixel_fovs</code>. More intuitive to name it that way.  </p>